### ISSUE TITLE
M> Build WeChat Official Account to Push RChain team progress in real time

### ISSUE THREAD

**Issue by rjl493456442** [rchain/bounties#258]
In China, WeChat Official Account is content publishing platform similar to **Medium**. WeChat users in China receive all relevant content push by subscribing to the Official Account.

Taking into account that in China, WeChat is by far the most active social software, with more than **900 million users**. Therefore, the promotion of RChain through WeChat Official Account will play a very good effect.

The Official Account should be built immediately and  follow-up needs to have a professional team to complete the operation of RChain.

Consider apply an Wechat Official Account need a lot of stuff to do, e.g. Prove the direct partnership between the official public and the RChain team, Provide operational team identification，Pay Official Account apply fee(It is not free). So i think $2000 budget is suitable and this task need to be done in 3 weeks. Follow-up operations require additional budget

The Wechat Official Account QRCode is shown below. RChain architecture chinese translation has been published and pushed to all subscribers. 

Subscription is strongly recommended.

- [x] Wechat Official account construction

- [x] Upload recent chinese translations

![wechat_offical_account](https://user-images.githubusercontent.com/5959481/35605550-6a35733c-0684-11e8-91c9-ef63d2a979dd.jpeg)

**Comment by Mervyn853** [comment]
@rjl493456442 @yyh1102 There is already an existing group managed by @sportshark , I believe the QQ Chat group is more active. I dont think we want multiple groups on the same platform as there will be unnecessary overlap. There's already 45 members in the Rchain WECHAT group and 700+ and growing in the QQ group. ​
Chinese QQ Group ID : 中国朋友欢迎加入RChain粉丝群，QQ群号: 530718666
![image](https://user-images.githubusercontent.com/33376108/35606020-b98c6910-0687-11e8-84e0-f102447ae1ca.png)

**Comment by yuhuanmocmoc** [comment]
Personally, I also like to get information on WeChat Official Account. It's exactly what we need do to promote RChain and it's technology in China.

**Comment by yuhuanmocmoc** [comment]
@Mervyn853 I think you should read this issue more carefully. @rjl493456442 advices to build the WeChat Official Account which is similar to Medium. Both in terms of quantity and quality, the WeChat Official Account is definitely one of the best place to promote RChain. QQ group is too outdate, you must concede that.

**Comment by Mervyn853** [comment]
Alright great. I think this is a great initative, might want to work with the members in QQ on this as they are very active in promoting RChain in China especially @sportshark and Mao

**Comment by linsheng9731** [comment]
Very good proposal, Chinese fans need a credible official news channel.

**Comment by rjl493456442** [comment]
@Mervyn853 Hey, other people who are also promoting RChain in China are strongly welcome! But you know QQ Group is out of date and Wechat Official Account is the most efficient way to promoting something. So @sportshark can join us and let's make chain much much more popular!

**Comment by rjl493456442** [comment]
@sportshark Thanks for your remind. I'll negotiate with RChain team whether official chinese public channel is proper or not. If not, i'll change it to community type!

**Comment by sportshark** [comment]
@rjl493456442 @lapin7 There are another  RChain WeChat public channel 
![tim 20180201120315](https://user-images.githubusercontent.com/20704995/35660589-e5b33a1c-0747-11e8-8de7-539fcafadbeb.jpg)
In fact, he has been running free of charge.

**Comment by yyh1102** [comment]
@sportshark I found that this WeChat public channel is registered on 31, Jan, which is later than the time this issue created. I'm afraid it's not a good thing that RChain has two public channels.

**Comment by rjl493456442** [comment]
@sportshark Yesterday we created a public account for promoting Rchain, and we are putting the technical architecture, pi calculus and other documents that we translated into operation on our public account. I think you have no reason to create a public account today. I think a public channel for Chinese users is enough, you do not need to create one.
In order to avoid confusion, please cancel it

**Comment by sportshark** [comment]
@yyh1102 @rjl493456442 
1. This WeChat public channel is not built by me. You can contact its manager
2, I don't agree with you. All the WeChat public channels built by Rchain fans are for the purpose of spreading RChain. I think WeChat public channels can make more people know and understand RChain.

**Comment by yuhuanmocmoc** [comment]
@sportshark  I also love free stuff, but free is always the most expensive. It is all about the quality, and everyone has a right to earn a fair price for your work performance.

**Comment by rjl493456442** [comment]
@sportshark No worry, i'll negotiate with that public channel manager. I agree with the @yuhuanmocmoc said, for me and my teammates we need a suitable payment to promoting RChain in China. After all, with economic incentives, we will urge us to do a better job. 
Please @lapin7  and @sportshark  believe us, promote RChain is our initial starting point, but the necessary funding is necessary.

**Comment by zsluedem** [comment]
actually....they are doing great with the Wechat public accounts if you enter their public accounts which @sportshark provides.

**Comment by yuhuanmocmoc** [comment]
@zsluedem This conclusion is a bit too casual, that public account just be created one day ago. All right, calm down please, it's depends on the official  requirements.

**Comment by rjl493456442** [comment]
@lapin7 I think there is no need to creat the same issues.
Issue #258 and #277 are the same in actual content, and  #275 is same to #274.
@Whisker17 (and others?) please don't always propose the same issue. We should not distribute our power to do the same thing

**Comment by Whisker17** [comment]
@rjl493456442  sorry to hear this
I didn't know you guys are translating this docs,and I can't open the link you put in the #274 so that I can't check that.Sorry again.But I've work for this doc for many time,and I finish translation by myself a week ago,I just want others to review it more precise so I didn't create a issue.I think we should communicate with each other more to avoid this condition

**Comment by rjl493456442** [comment]
Project Progress Report：
Now me and my teammates have completed the Official Account construction named `RChain Blackboard` or `RChain黑板报` in chinese.

We divided articles published in `RChain Blackboard` into two categories. The first type is an article about technology in the RChain project. The second type is the progress of the RChain team, including weekly content, Roadmap information and more.

Currently we have published 4 articles to it: (1) Rholang tutorial, (2) Mobile-process-calculi (3) RChain Architecture and (4) Recent project progress. Now @yuhuanmocmoc is in charged of the Official account operation.

In the next months, we will upload more chinese translation made by ourselves or find elsewhere. Besides we have a Wechat group, once the public number has been updated, we will push the content to our readers in time.

**Comment by sportshark** [comment]
Great！I'm longing for more people to know about RChain

**Comment by rjl493456442** [comment]
Update (9 March):

We upload 12 articles now and 6 for rchain architecture documents, 4 for rholang tutorial and 2 for team week reports.

In the next step, more technical and introductory videos will be uploaded. 

Thanks for the effort in the past week @watermelonZJ :)

**Comment by rjl493456442** [comment]
Update (27 March):
Dear all, near the end of the month, I will make a brief report on the work of our team on the operation of WeChat Official Account.

In the last month, we uploaded 18 articles totally, including 4 articles for `Mobile-process-calculi` serial, 7 articles for `Rchain architecture` serial, 4 articles for `Rholang tutorial` serials and 3 articles for `Team progress`.

In the past month, we added **131 new subscribers**, and the total number of subscribers reached **1180**, and the average reading of each article reached more than **500 people**.

In the next month, we will upload medium blog translation, developer video and other resources to the wechat official account, stay tuned.
